ktlint-logger/src/main/kotlin/com/pinterest/ktlint/logger/util/Any.kt

```diff
@@ -0,0 +1,5 @@
+package com.pinterest.ktlint.logger.util
+
+public inline fun <reified T : Any> Any?.safeAs(): T? = this as? T
+
+public inline fun <reified T : Any> Any?.cast(): T = this as T
```

Comment:
The logger module should not be abused to expose those functions. Better expose them in module `ktlint-rule-engine-core` (also see https://pinterest.github.io/ktlint/api/overview/) in package `com/pinterest/ktlint/rule/engine/core/api`. And, please provide a KDoc for each method.

Reply:
Done. I put them here cause I didn't notice `ktlint-ruleset-standard` depend on `ktlint-rule-engine-core`.